src/Perpetuum/EntityFramework/EntityFactory.cs

```diff
@@ -66,6 +66,43 @@ public Entity Create(EntityDefault entityDefault,IIDGenerator<long> idGenerator)
 
             return entity;
         }
-    }
-}
+        public Entity CreateForData(string definitionName, IIDGenerator<long> idGenerator) {
```

Comment:
Probably worth a clear docstring about how this is called for and only for the data-dumper...

That said... is there a way to write this as an extension method that lives in the data-dumper project?  That way this won't be accidentally called from any core server methods

Reply:
removed